src/views/v2/rsvp/ari/form/fields/meta.php

```diff
@@ -26,6 +28,10 @@
  */
 $has_meta = apply_filters( 'tribe_tickets_rsvp_has_meta', $rsvp, false );
 
+if ( $has_meta instanceof Tribe__Tickets__Ticket_Object ) {
```

Comment:
I'm sorry but I still don't understand this part. The filter is supposed to return a boolean, [but it's going to be removed from ET+](https://github.com/moderntribe/event-tickets-plus/pull/1048/files#diff-ff165ca7948958575c9a6aba3296b0cb42018c8d28d48060a09ed9f645e3f9b1L21). We could keep it for backwards compatibility though.

But, I feel the if and the comparison is wrong. The template is already receiving the `$rsvp` as `Tribe__Tickets__Ticket_Object`, and the filter is returning a bool (ideally). Wouldn't it make more sense to do something like this:

```
$has_meta = apply_filters( 'tribe_tickets_rsvp_has_meta', $rsvp, false );

if ( empty( $has_meta ) || empty( $rsvp->has_meta_enabled() ) ) {
	return;
}
```

Reply:
```suggestion
if ( ! $has_meta || $has_meta instanceof Tribe__Tickets__Ticket_Object ) {
```

Reply:
Yeah I added this for backcompat